_usage/security.md

```diff
@@ -57,21 +55,18 @@ installation:
 - `tempdb_dbo`
 - `tempdb_guest`
 
-These users are MS SQL-specific and try to mimic what the target database is
-supposed to look like. 
+Moreover, there will be a <code>*dbname*&lowbar;db&lowbar;owner</code>
+and a <code>*dbname*&lowbar;dbo</code> for every database you create in
+Babelfish.
 
-In addition to that, all users that are normally available in PostgreSQL are
-available as well:
+These roles are there to implement the SQL Server ownership structure.
+They are created and maintained by Babelfish and should not be used by
+the application or the end user.
 
-- `postgres`
-- `pg_monitor`
-- `pg_read_all_settings`
-- `pg_read_all_stats`
-- `pg_stat_scan_tables`
-- `pg_read_server_files`
-- `pg_write_server_files`
-- `pg_execute_server_program`
-- `pg_signal_backend`
+You can use `CREATE LOGIN` to create a new Babelfish login.  Babelfish logins
+are implemented as PostgreSQL login roles of the same name.
 
-These users are unchanged, and can be used just like in a normal PostgreSQL
-installation.
+Note that Babelfish [does not support](/docs/usage/limitations-of-babelfish)
+`CREATE USER`, and any login you create is automatically a user in all
+databases.  `CREATE ROLE` is also not supported, so you cannot use roles in
+Babelfish.
```

Comment:
After moving the sentence about login behavior up, I'd simplify the last of the content:

Note that Babelfish [does not support](/docs/usage/limitations-of-babelfish)
`CREATE USER` or `CREATE ROLE` syntax.

Reply:
Done.